smsVoIncident.js

```diff
@@ -0,0 +1,40 @@
+const qs = require('qs');
```

Comment:
maybe a file name of `createVoIncidentFromSms`?

Reply:
yes